content/user-guide/games/para.md

```diff
@@ -162,12 +171,13 @@ InputSetRAWAPI=0
 InputSetXInput=1
 ```
 
-_Please Note:_ If you are running a _hardened_ version of Linux, (SELinux/Gentoo/Some Arch variants), OutFox should set up and initialise your controller automatically. If it does not work, you probably have restrictive udev rules for input devices by default. Quit OutFox and try placing the following udev rule in `/etc/udev/rules.d` (in a file such as `70-ppp.rules`):
+_Please Note:_ If you are running a _hardened_ version of Linux (SELinux/Gentoo/Some Arch variants), OutFox may not set up and initialise your controller automatically. If so you probably have restrictive udev rules for unknown input devices. Quit OutFox and try placing the following udev rule in `/etc/udev/rules.d` (in a file such as `70-ppp.rules`):
 
 ```KERNEL=="hidraw*", ATTRS{idVendor}=="0507", ATTRS{idProduct}=="0011", TAG+="uaccess"```
 
 Note that the file name must come before `73-seat-late.rules`, as this is where `uaccess` is applied.
-##### _Thanks to MrCarter for contributing this information_
+
+##### _Thanks to [@Tatsh](https://github.com/Tatsh) and [@Scraticus](https://github.com/Scraticus) for contributing this information_
```

Comment:
drumbs added this one, I am credited at the bottom of the page :)

([TheNathannator](https://github.com/TheNathannator)